rotkehlchen/chain/evm/decoding/aave/v3/decoder.py

```diff
@@ -283,9 +283,11 @@ def _decode_interest(
                     )
                 ) > 0
             ):  # parse the mint amount and balance_increase
-                if i_log.topics[0] == BURN and (earned_token := get_single_underlying_token(earned_token)) is None:  # type: ignore[assignment]  # we ignore if None  # noqa: E501
-                    log.error(f'Failed to find underlying token for Aave v3 token {earned_token}. Skipping')  # noqa: E501
-                    continue
+                if i_log.topics[0] == BURN:
+                    if (underlying_token := get_single_underlying_token(earned_token)) is None:
+                        log.error(f'Failed to find underlying token for Aave v3 token {earned_token}. Skipping')  # noqa: E501
+                        continue
+                    earned_token = underlying_token  # type: ignore[assignment]  # we ignore if None
```

Comment:
type ignore is no longer neede here

```suggestion
                    earned_token = underlying_token
```